hipster-qemu.json

```diff
@@ -0,0 +1,88 @@
+{
+  "variables": {
+    "vm_name": "oi-hipster",
+
+    "oi_version": "20181023",
+
+    "iso_url": "http://dlc.openindiana.org/isos/hipster/20181023/OI-hipster-text-20181023.iso",
```

Comment:
Why do we use 20181023, not the latest?

Reply:
Because this PR is that old :) look at #4 if you want the latest